README.md

```diff
@@ -101,12 +102,13 @@ A curated list of Nintendo 64 development resources including toolchains, docume
 ### Actively Maintained
 
 * [cen64](https://github.com/n64dev/cen64) - A [cycle-accurate](https://retrocomputing.stackexchange.com/questions/1191/what-exactly-is-a-cycle-accurate-emulator) An emulator for Windows, Linux, and Mac. While currently not fast enough to play games at full speed, it aims for perfect emulation by emulating the hardware inside of the console down to the register-transfer level. Widely used to test ROMs in lieu of or before using real hardware.
-* [Project64](https://www.pj64-emu.com) ([GitHub](https://github.com/project64/project64)) ([with Debugger](http://shygoo.net/pj64d/)) - An open source high-level emulator for Windows and Android. It does not accurately emulate console low-level details, but rather focus on compatibility with commercial games. It cannot be used to develop homebrew software with SDKs different from libultra.
```

Comment:
Why were the last two sentences here removed? To my knowledge, this is still the case. PJ64 doesn't emulate low-level details, nor focuses on hardware accuracy, nor can it be used to develop homebrew with libraries like libdragon or likely anything written in Rust.

Reply:
Sorry, the rebase seemed to do some weird things, I'll fix that ASAP.